examples/nunchuk_simpletest.py

```diff
@@ -8,12 +8,15 @@
 nc = adafruit_nunchuk.Nunchuk(board.I2C())
 
 while True:
-    x, y = nc.joystick
-    ax, ay, az = nc.acceleration
-    print("joystick = {},{}".format(x, y))
-    print("accceleration ax={}, ay={}, az={}".format(ax, ay, az))
-    if nc.button_C:
+    joystick, buttons, acceleration = nc.values
```

Comment:
For the simpletest example, let's stick with how it currently is instead of using `values`. We can let the various mouse examples be the ones that demonstrate that usage.

Will need to change the button syntax if you remove the `button_C` and `button_Z` properties. But otherwise, I think this example can remain unchanged.